Code/Framework/AzToolsFramework/AzToolsFramework/Entity/EditorEntityContextBus.h

```diff
@@ -34,6 +34,7 @@ namespace AzToolsFramework
         : public AZ::EBusTraits
     {
     public:
+        static const bool LocklessDispatch = true;
```

Comment:
nitpick: this can be make constexpr
```suggestion
        static constexpr bool LocklessDispatch = true;
```